state/initialize.go

```diff
@@ -56,13 +55,13 @@ type InitializeParams struct {
 	// to apply.
 	NewPolicy NewPolicyFunc
 
-	// MongoInfo contains the information required to address and
-	// authenticate with Mongo.
-	MongoInfo *mongo.MongoInfo
+	// MongoSession is the mgo.Session to use for storing and
+	// accessing state data. The caller remains responsible
+	// for closing the session.
```

Comment:
for closing *this* session ??

(to distinguish from the copy that is made)

Reply:
done